moma_demos/piloting_demo/launch/sim.launch

```diff
@@ -1,81 +1,83 @@
-    <arg name="robot_namespace"         value=""/>
-    <arg name="enable_ekf"              value="true" />
-    <arg name="simulation"              value="true"/>
-    <arg name="mpc"                     value="false"/>
-    <arg name="publish_robot_state"     value="false"/>  <!-- Since we do this in the current launch for the full robot -->
-  </include>
+    <group ns="ocs2_mpc">
+        <param name="task_file" value="$(find moma_ocs2)/config/mpc/task_panda.info"/>
+        <param name="robot_description_ocs2" command="$(find xacro)/xacro $(find moma_description)/urdf/superpanda.urdf.xacro 
+            mpc_model:=true use_nominal_extrinsics:=true tool:='none'"/>
+        <param name="base_type" value="1"/>
+    </group>
 
-  <!-- Publish camera odometry tf-->
-  <node pkg="odometry_conversion" type="odometry_conversion_node" name="tracking_camera_odometry_conversion" output="screen"/>
+    <group ns="panda">
+        <group if="$(eval arg('tool') == 'panda_hand')">
+            <node name="gripper_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="franka_gripper" />
+        </group>
 
-  <group ns="ocs2_mpc">
-    <param name="task_file" value="$(find moma_ocs2)/config/mpc/task_panda.info"/>
-    <param name="robot_description_ocs2" command="$(find xacro)/xacro $(find moma_description)/urdf/superpanda.urdf.xacro 
-      mpc_model:=true use_nominal_extrinsics:=true tool:='none'"/>
-    <param name="base_type" value="1"/>
-  </group>
+        <group if="$(eval arg('tool') in ['robotiq_2f_85', 'robotiq_2f_toolset'])">
+            <node name="gripper_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="robotiq_2f_85_controller" />
+        </group>
 
-  <group ns="control">
-    <group if="$(eval arg('tool') == 'panda_hand')">
-      <node name="gripper_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="franka_gripper" />
-    </group>
+        <!-- start joint position trajectory controller -->
+        <rosparam file="$(find moma_gazebo)/config/panda_controllers.yaml" command="load" />
+        <rosparam file="$(find moma_gazebo)/config/gripper_controllers.yaml" command="load" />
 
-    <group if="$(eval arg('tool') in ['robotiq_2f_85', 'robotiq_2f_toolset'])">
-      <node name="gripper_controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="robotiq_2f_85_controller" />
+        <node name="arm_start_controller_spawner" 
+              pkg="controller_manager" 
+              type="spawner" 
+              output="screen" args="joint_state_controller effort_joint_trajectory_controller" />
+        <node name="arm_stop_controller_spawner" 
+              pkg="controller_manager" 
+              type="spawner"
+              output="screen" args="--stopped mpc_controller path_passthrough_controller"
+              launch-prefix="bash -c 'sleep 20; $0 $@' " />
```

Comment:
Because more often I had that mpc startup would overlap with the simulation setting up and the robot would start in a very weird state. Dont know yet how to handle that nicely